.github/workflows/chromatic-staging.yml

```diff
@@ -27,7 +27,7 @@ jobs:
           CHROMATIC_PROJECT_TOKEN: ${{ secrets.STAGING_CHROMATIC_PROJECT_TOKEN }}
           CHROMATIC_INDEX_URL: https://index.staging-chromatic.com
           LOG_LEVEL: debug
-      - uses: actions/upload-artifact@v2
+      - uses: actions/upload-artifact@v4
```

Comment:
You'll want to rebase, this workflow shouldn't exist!

Reply:
Ope!